src/main.jsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { render } from 'react-dom';
+import App from './components/App';
+import 'semantic-ui-css/semantic.min.css';
+import './css/header.css';
+import './css/sidebar.css';
+import './css/viewer.css';
+import './css/features.css';
+import { HashRouter, Route } from 'react-router-dom';
+import { CookiesProvider } from 'react-cookie';
+import { configureStore } from '@reduxjs/toolkit';
+import { connect, Provider } from 'react-redux';
+
+import dummyReducer, { setLoading } from './stores/dummy/Reducer';
+
+const store = configureStore({
+    reducer: {
+        dummyReducer: dummyReducer
+    }
+});
+
+const mapState = (state) => {
+    const { isLoading } = state.dummyReducer;
+    return {
+        isLoading
+    };
+};
+
+const actionCreators = {
+    setLoading
+};
+
+const ConnectedApp = connect(
```

Comment:
This should not be here

Reply:
Where would you expect to see this instead?

Reply:
I'd like it to be broken into separate files and/or directories. There's a lot going on here.